static/js/components/email/EmailCompositionDialog.js

```diff
@@ -43,32 +43,30 @@ export default class EmailCompositionDialog extends React.Component {
     }
   };
 
-  automaticCheckbox = () => {
-    const {
-      activeEmail: { inputs },
-      updateEmailFieldEdit,
-    } = this.props;
+  setAutomaticEmailType = (sendAutomaticEmails: boolean): void => {
+    const { updateEmailFieldEdit } = this.props;
+    updateEmailFieldEdit('sendAutomaticEmails', {
+      target: {
+        value: sendAutomaticEmails
+      }
+    });
+  }
 
-    return <Checkbox
-      label="Automatically send this message in the future, whenever new users join who meet these criteria"
-      className="email-automatic"
-      checked={inputs.sendAutomaticEmails || false}
-      value={true}
-      onCheck={e => {
-        updateEmailFieldEdit('sendAutomaticEmails', {
-          target: {
-            value: e.target.checked
-          }
-        });
-      }}
-    />;
-  };
+  renderAutomaticEmailSettings = (sendAutomaticEmails: boolean): React$Element<*> => (
+    <AutomaticEmailOptions
+      sendAutomaticEmails={sendAutomaticEmails}
+      setAutomaticEmailType={this.setAutomaticEmailType} />
```

Comment:
style nit: can you put the closing JSX bracket on it's own line? like

```js
<MultiLineComponent
  foo='foo'
  bar='bar'
/>
```

instead of

```js
<MultiLineComponent
  foo='foo'
  bar='bar' />
```